.github/workflows/build_and_deploy.yml

```diff
@@ -17,6 +17,11 @@ on:
     branches:
       - main
   workflow_dispatch:
+    inputs:
+      target_directory:
+        description: 'The directory to deploy the docs to'
+        required: true
+        default: 'dev'
```

Comment:
@Czaki one question here — what happens when you trigger manually? You get a small text field to fill in?

Reply:
Yes. You may play with the benchmark workflow on the main repo.

Reply:
Neat! ❤️